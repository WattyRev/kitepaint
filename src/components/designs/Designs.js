import React from "react";

const Designs = () => <div>Designs</div>;

export default Designs;
