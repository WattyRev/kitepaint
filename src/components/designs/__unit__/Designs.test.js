import React from "react";
import { shallow } from "enzyme";
import Designs from "../Designs";

describe("Designs", () => {
  it("renders", () => {
    expect.assertions(1);
    const wrapper = shallow(<Designs />);
    expect(wrapper).toMatchSnapshot();
  });
});
