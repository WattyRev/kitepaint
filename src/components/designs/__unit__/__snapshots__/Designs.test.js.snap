// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Designs renders 1`] = `
<div>
  Designs
</div>
`;
